src/apps/vpn/ui/ui.qrc

```diff
@@ -70,6 +70,7 @@
         <file>screens/ScreenViewLogs.qml</file>
         <file>screens/settings/appPermissions/AppPermissionsList.qml</file>
         <file>screens/settings/appPermissions/ViewAppPermissions.qml</file>
+        <file>screens/settings/InformationCard.qml</file>
```

Comment:
[non-blocking] What do you think about moving the InformationCard component to nebula? Seems like a re-usable component?

Reply:
I also thought about his. I left it here since it's only being used in settings files but when/if we need to use it elsewhere, we should def bump it to nebula.